ArcadiaParties.API/Controllers/CalendarController.cs

```diff
@@ -30,7 +30,15 @@ public CalendarController(IMediator mediator)
         [HttpGet("{id}")]
         public async Task<IActionResult> GetUsersOfDepartment(int id, CancellationToken cancellationToken)
         {
-            var query = new GetUsersOfDepartmentQuery(id);
+            var query1 = new DepartmentExistsQuery(id);
```

Comment:
query1? Couldn't you come up with some meaningful name?

Reply:
Renamed